src/article_metrics/api_v2_logic.py

```diff
@@ -167,3 +169,31 @@ def summary(page, per_page, order):
     start_pos = per_page * (page - 1) # slices are 0-based
     offset = start_pos + per_page
     return len(rows), rows[start_pos:offset]
+
+
+@cache(use=TWO_MIN_CACHE)
+def citations_by_version(msid, version):
+    doi = f"{utils.msid2doi(msid)}.{version}"
+    crossref_citations = crossref.parse(crossref.fetch(doi), doi)
+
+    if not crossref_citations:
+        raise Http404("article version not found")
```

Comment:
minor: not familiar with django best practices, but referring HTTP in the logic appears to be wrong scope. (i.e. should `logic` know about the transport layer?)

Reply:
The scope of things are a bit grey in here, but on reflection I agree with this, bumped it up.